pinery-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -832,6 +832,15 @@ public String extractStringValueFrom(ResultSet rs) throws SQLException {
       STAIN("stain", "Stain"),
       SLIDES("slides", "Slides"),
       DISCARDS("discards", "Discards"),
+      DISTRIBUTED("distributed", "Distributed") {
+        @Override
+        public String extractStringValueFrom(ResultSet rs) throws SQLException {
+          boolean distributed = rs.getBoolean("distributed");
```

Comment:
Should use `getSqlKey()` instead of repeating String `"distributed"` that gets set in the constructor